src/rules/noEmptyInterfaceRule.ts

```diff
@@ -48,7 +48,7 @@ function walk(ctx: Lint.WalkContext<void>) {
             (node.heritageClauses === undefined ||
              node.heritageClauses[0].types === undefined ||
```

Comment:
true, but this change of the type was only introduced recently with typescript@2.3

I'd like to enable `strict-type-predicates`, too. But there will be problems with some TypeChecker APIs that are not yet compatible with strictNullChecks